lib/t/feeds.ex

```diff
@@ -244,7 +236,6 @@ defmodule T.Feeds do
     |> not_liked_profiles_q(user_id)
     |> not_liker_profiles_q(user_id)
     |> not_seen_profiles_q(user_id)
-    |> not_expired_match_with_q(user_id)
```

Comment:
Would this change result in users seeing their old matches?

Reply:
@ruslandoga not really, this was a workaround and not a proper solution since expired matches were deleted from ExpiredMatch table (and correspondingly from this query) anyways upon "expired_match" being "seen"